library/alloc/src/vec/mod.rs

```diff
@@ -1765,10 +1765,6 @@ impl<T, A: Allocator> Vec<T, A> {
 
     /// Moves all the elements of `other` into `self`, leaving `other` empty.
     ///
-    /// # Panics
-    ///
-    /// Panics if the number of elements in the vector overflows a `usize`.
```

Comment:
It can overflow if you take two `let mut vec = Vec::<()>::new(); vec.resize(usize::MAX, ());`. In addition the resulting vec isn't allowed to be bigger than `isize::MAX` bytes.

Reply:
I changed the panic note to be the same as for `Vec::push`.